spec/correctness_spec.rb

```diff
@@ -167,6 +175,11 @@
             end
             Pet.import columns, all_values, :timestamps => false, :on_duplicate_key_update => columns
           end
+
+          # We need a better syntax to do this sort of stuff in tests
+          # fail Pet.inline_schema_config.instance_variable_get(:@ideal_indexes).inspect
+          Pet.inline_schema_config.instance_variable_set(:@ideal_indexes, Set.new)  unless UNIQUE_CONSTRAINT
```

Comment:
i feel like my ancient `active_record_inline_schema` gem is getting in the way here

in all my new libs / app work, i just use plain sql to create tables / change them

feel free to remove it - maybe see if that clarifies things?

Reply:
Let's get this release cut, then maybe do a big clean up and move to a version 2?